docker/Dockerfile.hsm

```diff
@@ -18,7 +18,8 @@ COPY --from=kms /usr/local/bin/step-kms-plugin /usr/local/bin/step-kms-plugin
 
 USER root
 RUN apt-get update
-RUN apt-get install -y --no-install-recommends pcscd libpcsclite1
+RUN apt-get install -y --no-install-recommends pcscd libpcsclite1 ca-certificates
+RUN update-ca-certificates
```

Comment:
```suggestion
```

Reply:
The post-install step for `ca-certificates` will update them, so I don't think `update-ca-certificates` needs to be run here.